tox.ini

```diff
@@ -53,7 +53,7 @@ commands =
     python run_pylint.py
 deps =
     pep8
-    pylint
+    -ehg+https://bitbucket.org/logilab/pylint#egg=pylint
```

Comment:
`hg+{URI}` tells `pip` to clone a mercurial repo at the given URI.

The `-e` is an artifact I don't fully understand, I was just parroting what we did for the upstream `httplib2` (when Py3 was failing):
https://github.com/GoogleCloudPlatform/gcloud-python/commit/218f6867beede16b22daddef7c4001ae305419b6